sql/data_model.sql

```diff
@@ -0,0 +1,43 @@
+BEGIN;
+
+-- add pgcrypto extension so that we can use the gen_random_uuid function.
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+
+DROP TABLE IF EXISTS formdata CASCADE;
+CREATE TABLE formdata(
+  id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+  domain text NOT NULL,
```

Comment:
some more random thoughts from doing this sizing exercise. we should be explicit in the number of bytes domain in the event that we index it. the index size will be number of rows * size of column so in a table that is large those extra bytes can add up